### ISSUE TITLE
lolcommits --devices no longer works

### ISSUE THREAD

**Issue by tooluser** [lolcommits/lolcommits#393]
```
Tamalpais:orders joshua$ lolcommits --devices

private method `executable_path' called for #<Lolcommits::CaptureMacVideo:0x00007f962bb162a0>
```

Sorry, I can't submit a PR now, I expect it's easy, just wanted to get a note in. Will if I can get a break!

**Comment by matthutchinson** [comment]
Will take a look shortly and fix/release if I can.

**Comment by matthutchinson** [comment]
Will be fixed in next release 0.15.1 (will push to Ruby gems shortly)
